src/databricks/labs/ucx/installer/policy.py

```diff
@@ -89,9 +90,10 @@ def _get_instance_pool_id(self) -> str | None:
 
     def _definition(self, conf: dict, instance_profile: str | None, instance_pool_id: str | None) -> str:
         latest_lts_dbr = self._ws.clusters.select_spark_version(latest=True, long_term_support=True)
+        node_type_id = self._ws.clusters.select_node_type(local_disk=True, min_memory_gb=16)
```

Comment:
@nkvuong : I receive the NotAvailableInRegion error when the `min_memory_gb=16` is set. What should I do?

Reply:
@JCZuurmond did you upgrade databricks-sdk-py locally? `make dev` should do that

Reply:
did the trick, thanks